lib/ngx/ssl.lua

```diff
@@ -93,7 +189,7 @@ function _M.set_der_cert(data)
         error("no request found")
     end
 
-    local rc = C.ngx_http_lua_ffi_ssl_set_der_certificate(r, data, #data,
+    local rc = ngx_lua_ffi_ssl_set_der_certificate(r, data, #data,
```

Comment:
style: the alignment of the arguments is off. Also, maybe now they could all fit on a single line? There are many places in this PR where this is the case as well